Celeste.Mod.mm/Mod/Everest/Everest.Content.cs

```diff
@@ -22,6 +23,7 @@ public sealed class AssetTypeXml { private AssetTypeXml() { } }
     public sealed class AssetTypeText { private AssetTypeText() { } }
     public sealed class AssetTypeLua { private AssetTypeLua() { } }
     public sealed class AssetTypeMetadataYaml { private AssetTypeMetadataYaml() { } }
+    public sealed class AssetTypeEverestIgnore { private AssetTypeEverestIgnore() { } }
```

Comment:
Like Kalobi mentioned, with #471 `.everestignore` would be blacklisted from ingest by default. We _could_ whitelist it, but as we never need it again after loading it in I think it'd be cleaner to just remove it from the pipeline